### ISSUE TITLE
Future Warning from H5Py

### ISSUE THREAD

**Issue by scottcoughlin2014** [gwpy/gwpy#722]
```
In [1]: from gwpy.timeseries import TimeSeries
   ...: data = TimeSeries.fetch_open_data('H1', 1126259446, 1126259478)
   ...: 
/Users/scottcoughlin/Documents/virtualenvs/GravitySpy-py27/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
```

Not a big deal but every time the `TimeSeries.get` or any of its derivatives are called you see this fun message. Just to be put on the radar.

I see this Future Warning with at least
`h5py (2.7.1)`

**Comment by duncanmmacleod** [comment]
@scottcoughlin2014, this is an issue related only to `h5py`, and not anything to do with gwpy. That said, this is also already fixed in h5py/h5py#985 (and possibly h5py/h5py#926 as well), so you should be able to update to h5py-2.8.0rc1 to remove the warning:

```
pip install "h5py==2.8.0rc1"
```

**Comment by scottcoughlin2014** [comment]
Ya no I just was curious. Thanks @duncanmmacleod
